e2e_tests/src/e2e_conduit/fixtures/processors/block_evaluator.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from e2e_conduit.fixtures.plugin_fixture import PluginFixture
+
+logger = logging.getLogger(__name__)
+
+
+class BlockEvaluator(PluginFixture):
+    def __init__(self, catchpoint=""):
+        self.catchpoint = catchpoint
+        super().__init__()
+
+    @property
+    def name(self):
+        return "block_evaluator"
+
+    def setup(self, accumulated_config):
+        try:
+            self.algod_data_dir = accumulated_config["algod_data_dir"]
```

Comment:
nit: usually object attributes are defined in the constructor. So consider adding `algod_data_dir`, `algod_token` etc. to `__init__()` as `None`'s or another sensible initial value.